dashboard/app/controllers/api/v1/json_api_controller.rb

```diff
@@ -5,4 +5,9 @@ class Api::V1::JsonApiController < ApplicationController
   rescue_from CanCan::AccessDenied do
     head :forbidden
   end
+
+  # Don't differentiate between missing and forbidden records
+  rescue_from ActiveRecord::RecordNotFound do
+    head :forbidden
+  end
```

Comment:
Good point. The user would have to be authenticated to hit the API in the first place, but that would mean all teachers. We might as well go safer and return the uniform code even if it's not totally needed.